patches/chrome-browser-ui-tabs-tab_strip_model.h.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/chrome/browser/ui/tabs/tab_strip_model.h b/chrome/browser/ui/tabs/tab_strip_model.h
+index f7282146e44d05a4f923adeed59089423bc6be62..1a92bed95ca16c45a982238c7cf5e78648125d40 100644
+--- a/chrome/browser/ui/tabs/tab_strip_model.h
++++ b/chrome/browser/ui/tabs/tab_strip_model.h
+@@ -384,8 +384,10 @@ class TabStripModel : public TabGroupController {
+   void CloseSelectedTabs();
+ 
+   // Select adjacent tabs
++  virtual
+   void SelectNextTab(
+       UserGestureDetails detail = UserGestureDetails(GestureType::kOther));
++  virtual
+   void SelectPreviousTab(
+       UserGestureDetails detail = UserGestureDetails(GestureType::kOther));
+ 
+@@ -512,6 +514,7 @@ class TabStripModel : public TabGroupController {
+   bool ShouldResetOpenerOnActiveTabChange(content::WebContents* contents) const;
+ 
+  private:
++ friend class BraveTabStripModel;
```

Comment:
To be extensible, please follow this pattern:
https://github.com/brave/brave-core/blob/master/patches/chrome-browser-content_settings-content_settings_manager_impl.h.patch
https://github.com/brave/brave-core/blob/master/chromium_src/chrome/browser/content_settings/content_settings_manager_impl.h